client/src/stories/0-Animation.stories.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import StartAnimation from '../components/StartAnimation';
+
+export default {
+  title: 'Animation',
```

Comment:
Add what component you are using with `title: 'Animation',
  component: StartAnimation,`